perception/object_recognition/detection/tensorrt_yolo3/CMakeLists.txt

```diff
@@ -132,7 +121,7 @@ if(TRT_AVAIL AND CUDA_AVAIL AND CUDNN_AVAIL)
     ${CUDA_LIBRARIES}
   )
 
-  add_library(tensorrt_yolo3_lib
+  ament_auto_add_library(tensorrt_yolo3_lib SHARED
```

Comment:
Why was this changed from static to shared?

Reply:
I'm not sure about the reason, but when I tried to compile with by static_library, I got the following error:
```
--- stderr: tensorrt_yolo3                                                                                                                                                                                      
/usr/bin/ld: libgpu_tensorrt_yolo3_lib.a(gpu_tensorrt_yolo3_lib_generated_YoloLayer.cu.o):(.data+0x0): multiple definition of `Yolo::yolo3'; libtensorrt_yolo3_lib.a(YoloLayer.cpp.o):(.data+0x0): first defined here
/usr/bin/ld: libgpu_tensorrt_yolo3_lib.a(gpu_tensorrt_yolo3_lib_generated_YoloLayer.cu.o):(.data+0x20): multiple definition of `Yolo::yolo2'; libtensorrt_yolo3_lib.a(YoloLayer.cpp.o):(.data+0x20): first defined here
/usr/bin/ld: libgpu_tensorrt_yolo3_lib.a(gpu_tensorrt_yolo3_lib_generated_YoloLayer.cu.o):(.data+0x40): multiple definition of `Yolo::yolo1'; libtensorrt_yolo3_lib.a(YoloLayer.cpp.o):(.data+0x40): first defined here
collect2: error: ld returned 1 exit status
```

Reply:
No problem